code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -260,7 +260,8 @@ Doesn't work on other aliens/AI.*/
 	var/list/structures = list(
 		"resin wall" = /obj/structure/alien/resin/wall,
 		"resin membrane" = /obj/structure/alien/resin/membrane,
-		"resin nest" = /obj/structure/bed/nest)
+		"resin nest" = /obj/structure/bed/nest,
```

Comment:
```suggestion
		"resin door" = /obj/structure/mineral_door/resin,
		"resin nest" = /obj/structure/bed/nest)
```
Put all the Tegu-only list things above the last thing, this is to avoid merge conflicts.
If /tg/ adds something in this file, it will end up like this:
"resin nest" = /obj/structure/bed/nest,)
"resin door" = /obj/structure/mineral_door/resin)
"resin ect" = /obj/structure/whatever)